Aquality.WinAppDriver/src/Aquality.WinAppDriver/Resources/settings.json

```diff
@@ -22,5 +22,8 @@
   },
   "logger": {
     "language": "en"
+  },
+  "elementCache": {
+    "isEnabled": true
```

Comment:
What if I want to cache some elements and do not cache other?

Reply:
Then you would define your custom element type and override property IElementCacheConfiguration CacheConfiguration in it with your custom configuration